src/item.h

```diff
@@ -383,6 +412,7 @@ class item : public visitable<item>
     * @param batch The batch crafting number to multiply data by
     */
     std::string info( bool showtext, std::vector<iteminfo> &dump, int batch ) const;
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
You can group declarations with the special comments
```C++
/**
 comment for function a, b and c together.
*/
/*@{*/
void a();
void b();
void c();
/*@}*/
```
There are many examples of this in this header. The comment will be used by doxygen to generate documentation.